ocs_ci/deployment/hub_spoke.py

```
@@ -4352,6 +4357,72 @@ def destroy_aws_hcp_cluster(self, timeout=1200):
 
         return verified
 
+    def _discover_oidc_config(self):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `_discover_oidc_config`

Add return type annotation: `None`

(ANN202)